site/docs/plugins/chat-members.md

```diff
@@ -0,0 +1,156 @@
+# Chat Members Plugin (`chat-members`)
+
+Automatically store information about users on a chat and retrieve it easily.
+
+## Introduction
+
+In many situations, it is necessary for a bot to have information about all the users of a given chat.
+Currently, though, the Telegram Bot API exposes no method that allows us to retrieve this information.
+
+This plugin comes to the rescue: automatically listening to `chat_member` events and storing each `ChatMember` objects.
+
+## Usage
+
+### Storing Chat Members
+
+You can use a valid grammY [storage adapter](https://grammy.dev/plugins/session.html#known-storage-adapters) or an
+instance of any class that implements the [`StorageAdapter`](https://deno.land/x/grammy/mod.ts?s=StorageAdapter) interface.
```

Comment:
1S/L. Also, use relative links. Also also, link to MD files not HTML files.
```suggestion
You can use a valid grammY [storage adapter](./session.md#known-storage-adapters) or an instance of any class that implements the [`StorageAdapter`](https://deno.land/x/grammy/mod.ts?s=StorageAdapter) interface.
```

Reply:
I badly missed these lmao